crypto/include/datatypes.h

```diff
@@ -157,13 +163,13 @@ void v128_left_shift(v128_t *x, int shift_index);
     ((((x)->v32[(bit) >> 5]) &= ~((uint32_t)1 << ((bit)&31))))
 
 /*
- * srtp_octet_string_is_eq(a, b, len) returns 1 if the length len strings
- * a and b are not equal. It returns 0 otherwise. The running time of the
+ * srtp_octet_string_is_eq(a, b, len) returns true if the length len strings
+ * a and b are NOT equal. It returns false otherwise. The running time of the
```

Comment:
consider renaming, srtp_byte_array_is_equal ?? ,  and making it more logical. This can be done in a new PR maybe to make it cleaner.

Reply:
Yeah. we should do that and make the name align with the result expected.